package.json

```diff
@@ -124,11 +124,15 @@
     "webextension-polyfill": "0.9.0"
   },
   "overrides": {
```

Comment:
please alphebetize so it's easier to see what changed here

Reply:
i have a slight preference to NOT update packages via overrides unless they are actually causing brave-core audit failures (many of these are ignored already via audit-config); otherwise we will need to keep updating overrides as the packages are updated.